src/ol_infrastructure/applications/edxapp/__main__.py

```diff
@@ -1269,6 +1272,159 @@ def cloud_init_user_data_func(
     ],
 )
 
+########################
+# Fastly CDN Managment #
+########################
+
+mfe_regex = "^/({})/".format("|".join(edxapp_mfe_paths))
+fastly_resource_options = {}
+if fastly_service_id := edxapp_config.get("fastly_service_id"):
+    fastly_resource_options = {
+        "protect": True,
+        "import_": fastly_service_id,
+        "ignore_changes": [
+            "backends",
+            "cache_settings",
+            "conditions",
+            "dictionaries",
+            "domains",
+            "name",
+            "snippets",
+        ],
+    }
+fastly_mitxonline_qa = fastly.ServiceVcl(
+    f"fastly-{stack_info.env_prefix}-{stack_info.env_suffix}",
+    backends=[
+        fastly.ServiceVclBackendArgs(
+            address=edxapp_mfe_bucket.bucket_domain_name,
+            name="MFE S3 Bucket",
+            override_host=edxapp_mfe_bucket.bucket_domain_name,
+            port=DEFAULT_HTTPS_PORT,
+            request_condition="MFE Path",
+            ssl_cert_hostname=edxapp_mfe_bucket.bucket_domain_name,
+            ssl_sni_hostname=edxapp_mfe_bucket.bucket_domain_name,
+            use_ssl=True,
+        ),
+        fastly.ServiceVclBackendArgs(
+            address=web_lb.dns_name,
+            name="AWS ALB for edxapp",
+            port=DEFAULT_HTTPS_PORT,
+            ssl_cert_hostname=edxapp_domains["lms"],
+            ssl_sni_hostname=edxapp_domains["lms"],
+            use_ssl=True,
+        ),
+    ],
+    cache_settings=[
+        fastly.ServiceVclCacheSettingArgs(
+            action="pass",
+            cache_condition="Django Admin Route",
+            name="Django Admin Route",
+        )
+    ],
+    comment="",
+    conditions=[
+        fastly.ServiceVclConditionArgs(
+            name="Django Admin Route",
+            statement='req.url ~ "^/admin"',
+            type="CACHE",
+        ),
+        fastly.ServiceVclConditionArgs(
+            name="MFE Path",
+            statement=f'req.url.path ~ "{mfe_regex}"',
+            type="REQUEST",
+        ),
+    ],
+    dictionaries=[
+        fastly.ServiceVclDictionaryArgs(
+            name="marketing_redirects",
+        )
+    ],
+    domains=[
```

Comment:
Do we not want the domain name to be pulled from the stack config?

Reply:
They should come from the config, I'll fix that